npm-shrinkwrap.json

```diff
@@ -280,85 +289,90 @@
         },
         "vary": {
           "version": "1.0.0",
-          "from": "vary@~1.0.0"
+          "from": "vary@1.0.0",
+          "resolved": "https://registry.npmjs.org/vary/-/vary-1.0.0.tgz"
         },
         "cookie": {
           "version": "0.1.2",
-          "from": "cookie@0.1.2",
+          "from": "https://registry.npmjs.org/cookie/-/cookie-0.1.2.tgz",
           "resolved": "https://registry.npmjs.org/cookie/-/cookie-0.1.2.tgz"
         },
         "merge-descriptors": {
           "version": "0.0.2",
-          "from": "merge-descriptors@0.0.2",
+          "from": "https://registry.npmjs.org/merge-descriptors/-/merge-descriptors-0.0.2.tgz",
           "resolved": "https://registry.npmjs.org/merge-descriptors/-/merge-descriptors-0.0.2.tgz"
         },
         "utils-merge": {
           "version": "1.0.0",
-          "from": "utils-merge@1.0.0",
+          "from": "https://registry.npmjs.org/utils-merge/-/utils-merge-1.0.0.tgz",
           "resolved": "https://registry.npmjs.org/utils-merge/-/utils-merge-1.0.0.tgz"
         }
       }
     },
     "logops": {
       "version": "0.2.0",
-      "from": "logops@0.2.0",
+      "from": "https://registry.npmjs.org/logops/-/logops-0.2.0.tgz",
       "resolved": "https://registry.npmjs.org/logops/-/logops-0.2.0.tgz",
       "dependencies": {
         "colors": {
           "version": "0.6.2",
-          "from": "colors@~0.6.2",
+          "from": "https://registry.npmjs.org/colors/-/colors-0.6.2.tgz",
           "resolved": "https://registry.npmjs.org/colors/-/colors-0.6.2.tgz"
         }
       }
     },
     "mongodb": {
       "version": "1.4.28",
-      "from": "mongodb@^1.4.7",
+      "from": "https://registry.npmjs.org/mongodb/-/mongodb-1.4.28.tgz",
       "resolved": "https://registry.npmjs.org/mongodb/-/mongodb-1.4.28.tgz",
       "dependencies": {
         "bson": {
           "version": "0.2.16",
-          "from": "bson@~0.2",
+          "from": "https://registry.npmjs.org/bson/-/bson-0.2.16.tgz",
```

Comment:
After a second look at this, `version` and `resolved` haven't changed, so it seems to be ok. NTC.